### DISCUSSION TITLE
Multi-frame rendering from CLI

### DISCUSSION THREAD

**Discussion by solanki21** [inlife/nexrender#954] [General]
What settings do I need in the Nexrender.json file to use multi frame rendering feature?
I see this statement in the docs:
multiFrames - boolean, providing true will attmpt to use aerender's built-in feature of multi frame rendering (false by default)

But it seems to be only available as Javascript API. Is there any setting for the JSON config file?

**Comment by inlife** [comment]
It's not a job-related (video-related) but rather a renderer-related setting. It is something you set when you launch either nexrender-cli or nexrender-worker binaries. Try to check the available options via `--help`. It should be mentioned in the list.
